app/views/blocks/campusQuestion.html

```diff
@@ -2,14 +2,18 @@
   <input
     type="text"
     ng-model="answer.value"
-    ng-model-options="{ debounce: 1000 }"
+    ng-model-options="{ debounce: 100 }"
     placeholder=""
     class="form-control"
+    show-errors
     ng-required="block.required"
     uib-typeahead="campus for campus in searchCampuses($viewValue)"
     typeahead-loading="loadingCampuses"
-    typeahead-min-length="4"
-    typeahead-wait-ms="300"
+    typeahead-min-length="2"
+    typeahead-wait-ms="100"
```

Comment:
Does that query ERT (and in turn, Infobase) 10x/sec? Ick.

Reply:
The typeahead wait time is how long between the time they type a character and when the query to ERT's API happens. It still only does one query per character max, in this case it would be upwards of 10x/sec if they are typing that much at that pace.

Reply:
@wrandall22 Would it be better to load all the campus names into an array when the form loads?

Reply:
I don't think downloading 5,000 or 25,000 campuses is the right direction to go.

You mention, `The main thing is that we want the user to see that it is an autocomplete box when they start typing.`

We can do a few things to achieve that:
- Add small text under the input stating it's an autocomplete box.
- On the initial input focus, show the autocomplete box with a list of the most popular schools people select.

Reply:
I'm fine with @dr-bizz proposal.

Reply:
@dr-bizz I'm not sure if we'd want to show popular schools. Right now it shows an error message when they type in something not in the list. So that may be enough to get them to realize it is an autocomplete box without needing to explicitly say that.

I will change it to 200.